ray-operator/apis/ray/v1/raycluster_webhook.go

```diff
@@ -87,3 +92,26 @@ func (r *RayCluster) validateWorkerGroups() *field.Error {
 
 	return nil
 }
+
+func (r *RayCluster) ValidateRayClusterSpec() *field.Error {
+	if r.Annotations[RayFTEnabledAnnotationKey] == "false" && r.Spec.GcsFaultToleranceOptions != nil {
+		return field.Invalid(
+			field.NewPath("spec").Child("gcsFaultToleranceOptions"),
+			r.Spec.GcsFaultToleranceOptions,
+			fmt.Sprintf("GcsFaultToleranceOptions should be nil when %s is disabled", RayFTEnabledAnnotationKey),
```

Comment:
`when %s is disabled` might be confusing.
```suggestion
			fmt.Sprintf("GcsFaultToleranceOptions should be nil when %s annotation is set to false", RayFTEnabledAnnotationKey),
```